api/src/main/java/com/codeforcommunity/ApiMain.java

```diff
@@ -45,7 +30,7 @@ public void startHttpsServerApi() {
 
     HttpServer server = vertx.createHttpServer(serverOptions);
     Router router = apiRouter.initializeRouter(vertx);
-    int port = Integer.parseInt(serverProperties.getProperty("server.port"));
+    int port = 5000;
```

Comment:
Shouldn't this be 8081? 

Also can you separate this to a private final constant